log/sublogger.go

```diff
@@ -38,17 +38,18 @@ func (sl *SubLogger) setLevels(newLevels Levels) {
 
 // getFields returns sub logger specific fields for the potential log job.
 // Note: Calling function must have mutex lock in place.
-func (sl *SubLogger) getFields() *logFields {
+func (sl *SubLogger) getFields() *fields {
 	if sl == nil || globalLogConfig == nil || globalLogConfig.Enabled == nil || !*globalLogConfig.Enabled {
 		return nil
 	}
-
-	fields := logFieldsPool.Get().(*logFields) //nolint:forcetypeassert // Not necessary from a pool
+	fields := logFieldsPool.Get().(*fields) //nolint:forcetypeassert // Not necessary from a pool
 	fields.info = sl.levels.Info
 	fields.warn = sl.levels.Warn
 	fields.debug = sl.levels.Debug
 	fields.error = sl.levels.Error
 	fields.name = sl.name
 	fields.output = sl.output
+	fields.botName = sl.botName
```

Comment:
now that the struct is called `fields`, its best to rn this and the other local variables